tests/e2e/configurer/config/constants.go

```diff
@@ -14,7 +14,9 @@ const (
 	// number of blocks it takes to vote for a single validator to vote for a proposal
 	PropVoteBlocks float32 = 1
 	// number of blocks used as a calculation buffer
-	PropBufferBlocks float32 = 8
+	PropBufferBlocks float32 = 30
+	// number of blocks used as a calculation buffer (used to set actual voting period)
+	PropBufferBlocksVotePeriod float32 = 8
```

Comment:
We now run multiple blocks a second both pre and post upgrade. The 30 prop buffer blocks are needed to allow enough time to pass for the upgrade prop for the upgrade height to not be in the past.